lib/graphql/schema/member/type_system_helpers.rb

```diff
@@ -35,4 +35,4 @@ def kind
       end
     end
   end
-end
+end
```

Comment:
Should this whitespace change be included, or can it be reverted?

Reply:
Yep, seems like I need to make some editor config changes!